### ISSUE TITLE
npm is crashing 

### ISSUE THREAD

**Issue by shanalishams** [npm/npm#18914]
[2017-08-12T21_48_34_707Z-debug.log](https://github.com/npm/npm/files/1402716/2017-08-12T21_48_34_707Z-debug.log)
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'install',
1 verbose cli   'gulp-an gular-filesort',
1 verbose cli   '--save-dev' ]
2 info using npm@5.3.0
3 info using node@v8.2.1
4 verbose npm-session ff3ca690001a3e0b
5 silly install loadCurrentTree
6 silly install readLocalPackageData
7 error cb() never called!
8 error This is an error with npm itself. Please report this error at:
9 error <https://github.com/npm/npm/issues>

**Comment by mreinstein** [comment]
I don't think this is a bug. I'm pretty sure this is being caused by an unintended space in the package name (i.e., 
you're doing `npm install gulp-an gular-filesort`,
 rather than `npm install gulp-angular-filesort`.)
